legal-api/tests/unit/models/__init__.py

```diff
@@ -130,6 +130,8 @@ def factory_business(identifier, founding_date=EPOCH_DATETIME, last_ar_date=None
                         fiscal_year_end_date=FROZEN_DATETIME,
                         legal_type=entity_type)
 
+    uow = versioning_manager.unit_of_work(db.session)
```

Comment:
Please add a comment here, as well as the description block
we use nested transactions and it is a known issue that multiple versions will have issues in test.
will help the next person if they are using this to create multiple version histories.